paasta_tools/contrib/check_orphans.py

```diff
@@ -262,13 +274,34 @@ def main() -> ExitCode:
         type=str,
         help="Comma separated list of services to ignore",
     )
+    parser.add_argument(
+        "--no-check-collisions",
+        default=False,
+        action="store_true",
+        help="Skip checking collisions",
+    )
+    parser.add_argument(
+        "--no-check-orphans",
+        default=False,
+        action="store_true",
+        help="Skip checking orphans",
+    )
```

Comment:
With apologies for the negatives, this is so we can keep the current invocation the same w/o affecting status quo